src/composables/useHotKey/index.js

```diff
@@ -26,7 +27,8 @@ function shouldIgnoreEvent(event) {
 }
 
 const eventHandler = (callback, options) => (event) => {
-	if (!!options.ctrl !== event.ctrlKey) {
+	const ctrlKeyPressed = isMac ? event.metaKey : event.ctrlKey
+	if (!!options.ctrl !== ctrlKeyPressed) {
```

Comment:
I do not like double negation for casting, what do you think about:

```suggestion
	if (ctrlKeyPressed !== Boolean(options.ctrl)) {
```

Reply:
Fine with that)